README.md

```diff
@@ -38,22 +38,26 @@ Wraps module into an ewasm-compatible constructor. It has two options:
 - memory deployer: wrap the module as a pre-defined memory section
 - custom section deployer: include the module as a custom section
 
-## CLI (WIP)
+## CLI
 
-`wasm-chisel` is available as a command line tool.
+`chisel` is available as a command line tool.
 
-It uses features implemented in the library as well in [wasm-gc] and [wasm-utils]. It comes with a configuration file `chisel.yaml`.
+It uses features implemented in the library as well in [wasm-gc] and [wasm-utils]. It comes with a configuration file `chisel.yml`.
 
-## Configuration file (WIP)
+`chisel` searches for `chisel.yml` in the current directory, if not specified otherwise using the flag `-c`.
+
+The only subcommand available at the moment is `chisel run`, which runs non-interactively according the the configuration file.
+
+## Configuration file
 
 The configuration file starts with a ruleset entry, where the name can be anything. Inside the ruleset are its options.
+It is important to note that the configuration parsing will not work if all the rules are prepended with a hyphen. Please avoid this until the configuration parser is generalized.
```

Comment:
I think it really has to keep "the hyphen", which makes it into a sequence. Having a mapping with duplicate names sounds really bad.

Reply:
Duplicate names should not be allowed in the first place.

Reply:
Why?

Reply:
no point in having a module be run twice.
also no point in having rulesets of the same name because they are addressed by name.

Reply:
Why? The module can have different configuration on two runs.

Reply:
Would rather defer this to a later PR in order to unblock you on RuneVM